src/main/java/uk/gov/hmcts/cmc/claimstore/ClaimStoreApplication.java

```diff
@@ -7,7 +7,7 @@
 
 @SpringBootApplication(exclude = DataSourceAutoConfiguration.class)
 @SuppressWarnings({"HideUtilityClassConstructor", "squid:S1118"}) // Spring needs a constructor, its not a utility class
-@EnableFeignClients
+@EnableFeignClients(basePackages = {"uk.gov.hmcts.cmc.claimstore", "uk.gov.hmcts.document"})
```

Comment:
Tim's comment is gone. Please use the `BASE_PACKAGE_NAME` constant from below to provide the scanning target. Does `uk.gov.hmcts.document` need to be scanned for Feign clients?

Reply:
Ignore, this cannot be done unfortunately, the annotation expects a constant value, apparently it has a problem with the base package being retrieved dynamically from the class.